diesel/src/expression_methods/global_expression_methods.rs

```diff
@@ -434,6 +434,80 @@ pub trait ExpressionMethods: Expression + Sized {
     fn asc(self) -> dsl::Asc<Self> {
         Asc::new(self)
     }
+
+    /// Converts this potentially nullable expression into one which will be **assumed**
+    /// to be not-null. This method has no impact on the generated SQL, however it will
+    /// enable you to attempt deserialization of the returned value in a non-`Option`.
+    ///
+    /// This is meant to cover for cases where you know that given the `WHERE` clause
+    /// the field returned by the database will never be `NULL`.
+    ///
+    /// This **will cause runtime errors** on `load()` if the "assume" turns out to be incorrect.  
+    /// If incorrect in `Nullable` tuples, this **will cause silent fails** (see second
+    /// example).
```

Comment:
I would skip this sentence, as:

* This is not a silent fail in my opinion, but a correct deserialisation
* That does not only happen for tuples